editor/editor_scale.cpp

```diff
@@ -32,12 +32,12 @@
 
 #include "core/os/os.h"
 
-static float scale = 1.0;
+float EditorScale::_scale = 1;
```

Comment:
We try to be explicit about float literals:
```suggestion
float EditorScale::_scale = 1.0f;
```